testing/jormungandr-testing-utils/src/testing/fragments/export.rs

```diff
@@ -35,22 +35,26 @@ impl FragmentExporter {
     pub fn read(&self) -> Result<Vec<Fragment>, FragmentExporterError> {
         self.read_as_bytes()?
             .iter()
+            .rev()
             .map(|bytes| Ok(Fragment::deserialize(bytes.as_ref()).unwrap()))
             .collect()
```

Comment:
Is it possible to just return an iterator here? Or would it be too troublesome to change the calling code?

Reply:
Oops, that `rev` is  a leftover actually